packages/analytics/analytics-chart/src/components/chart-plugins/VerticalLinePlugin.ts

```diff
@@ -2,6 +2,7 @@ import type { Chart, ChartType, Plugin, TooltipItem } from 'chart.js'
 
 interface VerticalLinePlugin extends Plugin {
   clickedSegment?: TooltipItem<ChartType>
+  pause?: boolean
```

Comment:
Added a way for the vertical line plugin to be "paused" so we don't render it while were doing a zoom selection